### ISSUE TITLE
Electricraft will not merge power into motors

### ISSUE THREAD

**Issue by cypren** [ReikaKalseki/Reika_Mods_Issues#60]
TL;DR: Merging power from two or more ElC sources into electric motors drains all of the sources but doesn't give any additional power to the motor.

I mentioned this in [a thread on the FTB forums](http://forum.feed-the-beast.com/threads/electricraft-reactorcraft-how-does-amperage-merging-splitting-work.80557/) but just got around to testing and confirming this problem in a clean world. The mod list installed in this world is very limited in order to ensure a clean test:
- CodeChickenCore 1.0.6.43
- NotEnoughItems 1.0.4.105
- DragonAPI V6f
- RotaryCraft V6f
- ReactorCraft V6e
- ElectriCraft V6a
- ChromatiCraft V6f
- ExpandedRedstone V6a
- GeoStrata V6a

Here's a very simple setup with one auroral battery running two electric motors:

![Simple Motors](http://d.pr/i/1dBFv+)

With the top battery's relay engaged, 268MW (65536V@4096A) is split to 134MW to each motor. So far, so good. Now we turn on the second relay, enabling a total of 536MW (65536V@8192A). The electric meter shows 4096A being delivered to each motor, but the dynamometer shows no change at all in power: both motors are still producing 134MW.

![Two Power Sources](http://d.pr/i/1jzjm+)

Disconnecting the second motor produces 268MW to the first motor, but this is still half of the 536MW that is being input into the network. Both batteries are being drained at full speed, and the extra 268MW appears to simply be getting wasted.

![One Motor](http://d.pr/i/1gzWu+)

This problem _appears_ to be specific to electric motors: using two auroral batteries to charge a graphene battery shows a very obvious doubling in the charge rate. As such, it's possible to work around the problem at the moment by wiring individual batteries in line with electric motors in order to allow them to merge power without wasting it, though this obviously makes power networks considerably more complicated.

Edited to add one more thing: this behavior appears to kick in any time that multiple sources are attached to the same power network. In the screenshot below, both relays are engaged, but the redstone current to the lower battery has been disengaged, stopping it from emitting power. The electric meter shows no power flowing through that link, but the meters on the merged line bizzarely show 0V. The electric motor on the end is producing 134MW -- half of what it should be getting from the one active battery on top. I've seen similar behavior from inductive generators as well when one is receiving shaft power and another is not. Whatever this problem is, it doesn't appear to be unique to batteries.

![One Power Source](http://d.pr/i/18HEk+)

**Comment by ReikaKalseki** [comment]
I see smoke coming from a motor in one screenshot. Are you trying to merge power into non-upgraded motors?

**Comment by cypren** [comment]
Thanks for the clarification. This would be the point where I go, "duhhhhh, you can upgrade motors?" and go look at the source code.

Okay, so I've gotten it working now. This must be the change from the patch notes way back where it talked about nerfing ElC to prevent bypassing shaft junction limits, right? So my understanding (based on a quick perusal of the code) is that motors can only take one input source unless given an inductive ingot, which allows them to take two, or a tungsten ingot, which lets them take four.

Is there something inside the game that hints at this, and I just completely missed it? Or is this just the kind of tribal knowledge that can only be acquired by reading the code or being told by someone else?

**Comment by ReikaKalseki** [comment]
I made it very clear back when I made the changes, and the changelogs indicated it. I get the feeling you only updated (and from a very old version) recently, making that useless to you.

**Comment by cypren** [comment]
Yes, this is my first time playing with ElC in quite a while, and certainly my first time doing anything intensive with it (moving large amounts of power from ReC).

I realize you've probably got more important things to work on yourself, but if I took the time to write a patch to add an ElC manual in the style of the RC manual, is it something that you'd consider including? I appreciate that ElC started as a very simple add-on to RC, but it feels like it has enough complexity and "gotchas" at this point that it warrants having in-game documentation.

**Comment by ReikaKalseki** [comment]
I would not be opposed to it, though it might function better as an "ElC addon" rather than code to add to ElC. You can look to how ReC does it to see how that would be done. If you have the java knowledge, it should not be difficult.
